src/screens/AddTrip/AddTrip.tsx

```diff
@@ -65,7 +65,8 @@ export const AddTrip = (props: Props): JSX.Element => {
                         {
                             name: Routes.SHARE_TRIP,
                             params: {
-                                trip: result.data?.createTrip,
+                                tripId: result.data?.createTrip.id,
+                                // mode: Mode.SHARE_TRIP,
```

Comment:
Is this an optional param that has intentionally been commented out?

Reply:
No we thought that we will need it, but realised that we can set the mode inside the ShareTrip.tsx file depending on weather a tripId was passed or not. So I have removed it